CONTRIBUTING.md

```diff
@@ -1,3 +1,85 @@
 # Contributing
 
-Comming soon ...
+Welcome and thank you for considering contributing to Trdelnik open source!
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+It also communicates that you agree to respect the time of developers managing and developing Trdelnik project. In return, we will
+reciprocate that respect by addressing your issue, assessing changes and help you finalize your PRs.
+
+## Table of Contents
+
+- [Code of Conduct](./CODE_OF_CONDUCT.md)
+- [Getting Started](#getting-started)
+  - [Issues](#issues)
+  - [PRs](#prs)
+- [Getting Help](#getting-help)
+
+## Code of Conduct
+
+We take our open source and community seriously and hold ourselves and other contributors to high standards of communication.
+By participating and contributing to this project, you agree to uphold ou [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+## Getting Started
+
+Contributors are made to this repository via issues and pull requests (PRs). A few general guidelines that cover both:
+
+- Search for existing issues and pull requests before creating your own
+- We work hard to make sure issues are handled in a timely manner but, depending on the impact, it could take a while to investigate
+  the root cause. A friendly ping in the comment thread to the submitter or a contributor can help draw attention, if your issue is blocking.
```

Comment:
(few changes) "We work hard to ensure issues are handled promptly but depending on the impact, it could take a while to investigate the root cause. A friendly ping in the comment thread to the submitter or a contributor can help draw attention if your issue is blocked."

Reply:
:heavy_check_mark: